### PR TITLE
Provider reference number

### PR DESCRIPTION
**Pull request by cleargif**
PT:
https://www.pivotaltracker.com/story/show/149998891

Why are we doing this?
- As a user I want to know what the provider ref number is.

What this PR does:
- Hint text inserted on the field to set context
- housekeeping of labels in general
- `.form-label`  missing from a few

### CODE REVIEW

File: app/views/external_users/claims/case_details/_fields.html.haml [hunk 1]
```diff
@@ -20,25 +20,22 @@
...
-        .form-col
-          = f.adp_text_field :providers_ref, label: t('common.external_user.providers_ref'), errors: @error_presenter
+    .form-row
+      .form-col.forced-width
+        = f.adp_text_field :providers_ref, label: t('common.external_user.providers_ref'), errors: @error_presenter, hint_text: t('common.external_user.providers_ref_hint')
 
-    - else
-      .form-row
-        .form-col
-          = f.adp_text_field :providers_ref, label: t('common.external_user.providers_ref'), errors: @error_presenter
```
Comment (cleargif) [inline app/views/external_users/claims/case_details/_fields.html.haml:32]: @colinbruce @jsugarman 

This is the only thing of interest really. I removed the `else` as the input is the same for both states.

Comment (jsugarman) [reply app/views/external_users/claims/case_details/_fields.html.haml:32]: looks sensible to me 👍

File: app/views/case_workers/admin/allocations/_scheme_filters.html.haml [hunk 2]
```diff
@@ -5,10 +5,10 @@
...
       %input#scheme_agfs{:name => "scheme", :type => "radio", :value => "agfs", :checked => "checked"}/
-      %label{:for => "scheme_agfs"} AGFS
+      %label.form-label{:for => "scheme_agfs"} AGFS
     .multiple-choice
       %input#scheme_lgfs{:name => "scheme", :type => "radio", :value => "lgfs"}/
-      %label{:for => "scheme_lgfs"} LGFS
+      %label.form-label{:for => "scheme_lgfs"} LGFS
 
     %noscript
       = submit_tag "Submit"
```
Comment (jsugarman) [inline app/views/case_workers/admin/allocations/_scheme_filters.html.haml:14]: rubocop might complain about no line break

**Review by jsugarman — APPROVED** [review]
👍

**Pull request merged**